dash/include/dash/GlobDynamicMem.h

```diff
@@ -1015,6 +906,60 @@ class GlobDynamicMem
   }
 
 private:
+
+  /**
+   * Native pointer of the final address of the local memory of
+   * a unit.
+   */
+  local_iterator update_lend(
```

Comment:
Technically it's not a setter. I refined the implementation and hopefully now it's clear that it's updating the internal state of the object.